### ISSUE TITLE
Secondary settings button missing descriptive text

### ISSUE THREAD

**Issue by anevins12** [WordPress/gutenberg#7203]
**Describe the bug**
The secondary settings button has "More" as its `aria-label`, but this does not describe what it represents and its action.

**To Reproduce**
1. Edit a page
2. View the markup of the three dotted settings icon beside the cog icon.

![example](https://user-images.githubusercontent.com/2318561/41094093-f7f301c0-6a44-11e8-8c37-6cc311ab8d3a.png)

**Expected behavior**
* The label of the button should describe what it is and what it does.
* That text should change when the action changes (when it has been expanded).

For example, the label could be changed from "More" to "Show additional settings" and "Hide additional settings" when content has been expanded.

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser: Chrome
 - Version: Latest
